### ISSUE TITLE
Bad output for `convox env` on a new app

### ISSUE THREAD

**Issue by mattmanning** [convox/rack#41]
```
$ convox apps create test3
Creating app test3... CREATING

$ convox env --app test3
<?xml version="1.0" encoding="UTF-8"?>
<ListBucketResult xmlns="http://s3.amazonaws.com/doc/2006-03-01/"><Name>env</Name><Prefix></Prefix><Marker></Marker><MaxKeys>1000</MaxKeys><IsTruncated>false</IsTruncated><Contents><Key>contact_us_for_a_quote</Key><LastModified>2014-03-03T15:07:22.000Z</LastModified><ETag>&quot;dc3796df991d4fa2b1ba3303dc8d4ec8&quot;</ETag><Size>17</Size><StorageClass>STANDARD</StorageClass></Contents><Contents><Key>index.html</Key><LastModified>2014-02-28T13:01:18.000Z</LastModified><ETag>&quot;d41d8cd98f00b204e9800998ecf8427e&quot;</ETag><Size>0</Size><StorageClass>STANDARD</StorageClass></Contents><Contents><Key>your_objects_here</Key><LastModified>2014-03-03T15:07:22.000Z</LastModified><ETag>&quot;d41d8cd98f00b204e9800998ecf8427e&quot;</ETag><Size>0</Size><StorageClass>STANDARD</StorageClass></Contents></ListBucketResult>
```

**Comment by mattmanning** [comment]
This only happens when the app is the "creating" state.

**Comment by joonathan** [comment]
Also, while app is still in creating state following could happen:

Updating environment... ERROR: AccessDenied: Access Denied
	status code: 403, request id:

**Comment by nzoschke** [comment]
Confirmed that the XML output is still happening.

**Comment by nzoschke** [comment]
Fixed on #631. Thanks @MiguelMoll !
